Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ReflectionAdapter.cpp

```diff
@@ -491,8 +659,10 @@ namespace AZ::DocumentPropertyEditor
                 auto containerAttribute = attributes.Find(Reflection::DescriptorAttributes::Container);
                 if (containerAttribute && !containerAttribute->IsNull())
                 {
+                    m_containers.SetValue(
+                        m_builder.GetCurrentPath(), AZStd::move(BoundContainer::CreateBoundContainer(access.Get(), attributes)));
```

Comment:
AZStd::move() can be removed here as well

Reply:
Got this with the one above.